src/deployers/PluginsDeployerFactory.h

```diff
@@ -45,7 +46,8 @@ namespace linuxdeploy {
                                                 boost::filesystem::path qtLibexecsPath,
                                                 boost::filesystem::path qtInstallQmlPath,
                                                 boost::filesystem::path qtTranslationsPath,
-                                                boost::filesystem::path qtDataPath);
+                                                boost::filesystem::path qtDataPath,
+                                                int qtMajorVersion = 5);
```

Comment:
You seem to have missed this occurrence.

Reply:
ok, changed.